src/main/java/com/hillel/items_exchange/controller/AdvertisementController.java

```diff
@@ -114,6 +116,29 @@
         return ResponseEntity.unprocessableEntity().body(HttpStatus.CONFLICT);
     }
 
+    @PostMapping("/setDefaultImage/{advertisementId}/{imageId}")
```

Comment:
URL doesn't use camel case and ``set`` into the body. Make it simpler and according to best practices: ```default-image``` and that will be enough.